rust/processor/src/db/common/models/coin_models/coin_utils.rs

```diff
@@ -324,6 +342,12 @@ impl CoinEvent {
             "0x1::coin::DepositEvent" => {
                 serde_json::from_str(data).map(|inner| Some(CoinEvent::DepositCoinEvent(inner)))
             },
+            "0x1::coin::CoinWithdraw" => {
+                serde_json::from_str(data).map(|inner| Some(CoinEvent::WithdrawCoinEventV2(inner)))
+            },
+            "0x1::coin::CoinDeposit" => {
+                serde_json::from_str(data).map(|inner| Some(CoinEvent::DepositCoinEventV2(inner)))
+            },
```

Comment:
why are we changing coin events? are they also affected?

Reply:
not really. I just checkout the file from the other PR. I can prune those.